resources/views/email.blade.php

```diff
@@ -202,23 +206,25 @@
                 <footer>
                     <div class="footer-top">
                         <div class="footer-logo">
-                            <img src="{{ asset('identification/logo.png') }}" alt="Blumilk logo"/>
+                            <a href="https://blumilk.pl" class="social-link" target="_blank">
```

Comment:
I would like to replace 
`href="https://blumilk.pl"`
with
`href="{{ config(app.url) }}"`

Reply:
Fixed